addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -85,6 +85,12 @@ const RatingPopupComposer = publicWidget.Widget.extend({
         // Instantiate the "Portal Composer" widget and insert it into the modal
         this._composer = new PortalComposer(this, this.options);
         return this._composer.appendTo(this.$('.o_rating_popup_composer_modal .o_portal_chatter_composer')).then(() => {
+            // Set border around the textarea instead of the portal composer body
+            const textarea =  this.el.querySelector('.o_portal_chatter_composer_body > div textarea[name="message"]');
+            if (textarea) {
+                textarea.parentElement.classList.remove('border');
+                textarea.classList.remove('border-0');
+            }
```

Comment:
Looks rather dirty and will not cleanly adapt to future changes on the template structure.

Could we instead remove the "border-0" on the textarea element and add scss rule on `.o_portal_chatter_composer_body textarea { border: 0 };`.

And then override this rule in the "portal_rating" module with the appropriate behavior?

(Same idea if we also need to adapt the "parent").

Reply:
Hello @awa-odoo 

Thanks for the suggestions and it works.
I have put the scss rule inside the `.o_portal_chatter` because when the view is in the full screen
and when we are on the pop-up screen `o_portal_chatter` class is not present so this `.o_portal_chatter_composer_body textarea { border: 0 };` rule is not applied.

so I don't think we need to override the rule in **portal_rating** module 

is this the correct approach correct me if i am wrong :slightly_smiling_face: 

Thanks

Reply:
That's even better, thanks a lot for the research 😄